scripts/kind-build-test.sh

```diff
@@ -264,7 +264,9 @@ fi
 # run e2e tests
 export SKIP_PYTHON_TESTS
 export RUN_PYTEST_LOCALLY
+set +e
 $SCRIPTS_DIR/run-tests.sh $AWS_SERVICE
+EXIT_CODE=$?
 
```

Comment:
-e after this, so if something else fails it gets caught

Reply:
The only thing after this is dumping the controller logs. If that fails, we still want to continue execution until we reach the `exit $EXIT_CODE` line.

Reply:
sounds ok for now, if the actual tests passed and dumping logs fail, let the build pass